modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -1113,15 +1193,17 @@ class NDB_Form_Statistics extends NDB_Form
     /**
      * GetSingleReliabilityData function
      *
-     * @param string $Param_Project the value of Param_Project
-     * @param string $Param_Site    the value of Param_Site
-     * @param string $name          the value of name
-     * @param string $inst_name     the value of inst_name
-     * @param string $cutoff        the value of cutoff
+     * @param string $Param_Project   the value of Param_Project
+     * @param string $suproject_query the value of suproject_query
```

Comment:
same typo? (and comment about fixing the others as long as you're fixing this one.)